code/__defines/gamemode.dm

```diff
@@ -51,9 +51,18 @@
 #define MODE_VAMPIRE "vampire"
 #define MODE_THRALL "thrall"
 
-
 #define DEFAULT_TELECRYSTAL_AMOUNT 25
 
+// Defines (NOT FLAGS) for making secret, random, and mixed secret less snowflake-string
+// and number dependant.
+#define ROUNDTYPE_SECRET       1
+#define ROUNDTYPE_MIXED_SECRET 2
+#define ROUNDTYPE_RANDOM       3
```

Comment:
Why can't these all be converted to strings so you don't need two defines? Doesn't look like they need to be numbers.